src/java.desktop/unix/classes/sun/awt/UNIXToolkit.java

```diff
@@ -500,12 +491,6 @@ public static GtkVersions getEnabledGtkVersion() {
                 new GetPropertyAction("jdk.gtk.version"));
         if (version == null) {
```

Comment:
`null` check for version can be removed as if version is not 3.... `GtkVersions.ANY` is returned.

Reply:
Updated.